aiqc/__init__.py

```diff
@@ -2977,10 +2968,15 @@ def from_encoderset(
 		# 2. Validate the lists of dtypes and columns provided as filters.
 		if (dataset_type == "image"):
 			raise ValueError("\nYikes - `Dataset.dataset_type=='image'` does not support encoding Feature.\n")
-		
-		sklearn_preprocess, only_fit_train, is_categorical = Labelcoder.check_sklearn_attributes(
-			sklearn_preprocess, is_label=False
-		)
+		elif (dataset_type == "text"):
```

Comment:
Did you try running these feature extractors on any Tabular datasets? In the future, if you really wanted to you could check that the `matching_columns` were of `dtype=='object'`

Reply:
not yet, but yes, if you think "object" dtype is a good way to identify text features then this already provides a way to phase out text dataset_type, probably use a default encoder in case the user doesn't specify one?

Reply:
good idea. i have not experimented with default encoders, but since we record the dtype it should be easy enough. it would take some validation: e.g. running a word count on a string-based input to determine if it is a category or a sentence/paragraph. Same thing with integers: >= 10 integers defaults to ordinal